nss-cohort-36/bangazon-api-virtual_magicians
# Jg product form

# Description
API file needed the create function and now the form does a create function on postman

Closes #[Enter issue #]

## Type of change
Please delete options that are not relevant.

# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout jg-product-form`
4. Run your API server python manage.py runserver
5.Open Postman
6.Make a POST request under localhost:8000/products
7.Put the following in the body:
{
    "name": "Motor",
    "price": 1,
    "description": "It's broken. I wouldn't buy it tbh...",
    "quantity": 1,
    "location": "Nashville",
    "created_at": "2020/02/26",
    "image_path": "123.com",
    "customer_id": 1,
    "product_type_id": 1
}
You should now have 
an object that has these fields above 🗡 

# Checklist:
